client/packages/config/src/config.ts

```diff
@@ -0,0 +1,21 @@
+import { Capacitor } from '@capacitor/core';
+
+// For production we get URL relative to web app
+const { port, hostname, protocol } = window.location;
```

Comment:
It does mean that we can only ever run the web app through electron, and I can see the scenario where a client has a hosted server with a real domain name, in which case they could use a web browser to connect to it. In fact we have had a lot of RFPs asking for a web solution - given that this is only a config option away, I think we should try to support it.

Reply:
This will work out of the box for server hosted on real domains (without any need to change config.js)